src/main/kotlin/org/taktik/freehealth/middleware/web/controllers/TarificationController.kt

```diff
@@ -89,7 +89,7 @@ class TarificationController(val tarificationService: TarificationService, val m
         guardPostNihii = guardPostNihii,
         guardPostSsin = guardPostSsin,
         anatomy = anatomy,
-        relatedService =  relatedService).let { mapper.map(it, TarificationConsultationResult::class.java) } }
+        relatedServices =  relatedServices).let { mapper.map(it, TarificationConsultationResult::class.java) } }
```

Comment:
Here put when :

1. relatedService is null : codes.map { null }
2. relatedService contains no ',' : codes.map { relatedService }
3. relatedService contains ',' : relatedServices.split(",").also { if (it.length != codes.length)  throw IllegalArgumentException("bla bla") }

Reply:
Hello @aduchate, thanks for your feedback. I adjusted the code accordingly

Reply:
I also provided a similar fix for the anatomy field which requires a single tooth per code